src/EA8AE0.c

```diff
@@ -44,7 +45,28 @@ INCLUDE_ASM(s32, "EA8AE0", func_8024140C_EA9EEC);
 
 #include "world/common/AddPlayerHandsOffset.inc.c"
 
-INCLUDE_ASM(s32, "EA8AE0", func_802417AC_EAA28C);
+// INCLUDE_ASM(s32, "EA8AE0", func_802417AC_EAA28C);
+
+
+extern s32 D_80243DD8_EAC8B8;
+s32 D_80243DDC_EAC8BC = 0;
```

Comment:
As discussed in discord, please remove this line setting this var to 0 and wrap the function in NON_MATCHING